pisa/stages/simple_fit/pi_simple_signal.py

```diff
@@ -16,7 +16,7 @@
 from pisa.core.bin_indexing import lookup_indices
```

Comment:
It might be better to move this to `/stages/data` as that's the place where we have other dummy data stages

Reply:
sounds good, though I noticed that the README documentation about this folder is quite out of date. Do you mind if I make a few changes there too?

Reply:
Be absolutely my guest!